torch/csrc/distributed/rpc/init.cpp

```diff
@@ -168,6 +166,16 @@ PyObject* rpc_init(PyObject* /* unused */) {
         return pyRemotePythonUdf(agent, dst, pickledPythonUDF, tensors);
       });
 
+  module.def(
+      "get_rpc_timeout",
+      []() { return RpcAgent::getDefaultRpcAgent()->getRpcTimeout(); },
+      R"(
+          Retrieve the timeout for all RPCs that was set during RPC initialization.
```

Comment:
I am guessing this will bring s lot of spaces in the front?

Should we start from the beginning of a line?

https://github.com/pytorch/pytorch/blob/3bc014ecf2a1badee5b1aa19e8b99273134f9590/caffe2/sgd/learning_rate_op.cc#L16

Reply:
@xush6528 it looks like it is autoformatted, if you build this PR and do `import torch.distributed.rpc as rpc ; help(rpc.get_rpc_timeout)` the format looks fine. Though I will check how this appears on the docs.